LibraryApp/src/app/screens/EmptyScreen/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import styles from './styles';
+
+function EmptyScreen() {
```

Comment:
whats this for?

Reply:
at that point the card said that it should navigate to an empty screen, and I took it very literal 😂